apps/balancer-tools/src/app/apr/round/(components)/PoolListTable.tsx

```diff
@@ -0,0 +1,156 @@
+import { ChevronDownIcon } from "@radix-ui/react-icons";
+import { Suspense } from "react";
+
+import Table from "#/components/Table";
+import votingGauges from "#/data/voting-gauges.json";
+import { Pool } from "#/lib/balancer/gauges";
+import { formatNumber } from "#/utils/formatNumber";
+
+import BalancerAPI from "../../(utils)/balancerAPI";
+import calculateRoundAPR from "../../(utils)/calculateRoundAPR";
+import { getBALPriceByRound } from "../../(utils)/getBALPriceByRound";
+import { getPoolRelativeWeight } from "../../(utils)/getRelativeWeight";
+import { Round } from "../../(utils)/rounds";
+import ClientSideBodyRow from "./ClientSideBodyRow";
+
+export default async function PoolListTable({ roundId }: { roundId: string }) {
+  return (
+    <div className="overflow-hidden rounded-lg w-full text-white">
+      <Table color="blue" variant="solid" shade="dark">
+        <Table.HeaderRow>
+          <Table.HeaderCell>Name</Table.HeaderCell>
+          <Table.HeaderCell>TVL</Table.HeaderCell>
+          <Table.HeaderCell> Votes % </Table.HeaderCell>
```

Comment:
```suggestion
          <Table.HeaderCell>Votes %</Table.HeaderCell>
```
what does this mean though? The percentage of total votes that pool has got in that round? In that case would rename it to `Voting %`